src/App.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Header from './components/Header/Header';
```

Comment:
I don't like `/Header/Header` in the import path. Please, create `index.js` file in every component folder and export root component. So, after that you will import component using `'./components/Header';` path